internal/veracode-api-wrapper/update-mitigation-info/README.md

```diff
@@ -0,0 +1,57 @@
+# Veracode API Wrapper - Update Mitigation Info
+
+Calls the Veracode API to update the mitigation comment of a build ID. This is effectively used in updating the comment in Veracode which is maintained in CSV files.
+
+---
+
+## Prerequisite
+
+### Prerequisites specific to Veracode 
+- Veracode Subscription
+- User role with "Mitigation" and "Mitigation and Comments"
+
+### Prerequisites specific to program 
+- <a href="https://github.com/wso2/security-tools/blob/master/internal/veracode-api-wrapper/updatemitigationinfo/requirements.txt">requirements.txt</a>
+
+---
+
+## Local Installation
+
+```
+$ virtualenv veracodeApiWrapper
+$ . env/bin/activate
+(veracodeApiWrapper) $ python updatemitigationinfo.py -h
+```
+
+---
+
+### Help
+
+```
+usage: updatemitigationinfo.py [-h] [-a APPLICATIONID] [-b BUILDID]
+                               [-u USERNAME] [-cf CSVFILEPATH]
+                               [-of OUTPUTFILEPATH]
+
+Veracode API wrapper to update mitigation comment.
+
+optional arguments:
```

Comment:
After the change this needs to be updated IMO. Now there should be some required args.

Reply:
Resolved.